Vagrantfile

```diff
@@ -152,7 +152,8 @@ EOF
         source /etc/environment
         source /etc/profile.d/sh.local
         set -eux -o pipefail
-        ${GOPATH}/src/github.com/containerd/containerd/script/setup/install-cni
+        cd ${GOPATH}/src/github.com/containerd/containerd
+        script/setup/install-cni
```

Comment:
Was this change still needed with the last updates of the script? (I expect them to be equivalent)

Reply:
It needs to be modified here, `go list` is in current directory or any parent directory to found `go.mod` file. e.g.

```shell
$ cd test # test directory is empty
$ go list -f -json
go: go.mod file not found in current directory or any parent directory; see 'go help modules'
```